core/src/test/java/org/testcontainers/utility/DirectoryTarResourceTest.java

```diff
@@ -68,4 +104,31 @@ public void simpleRecursiveClasspathResourceTest() throws TimeoutException {
         // ExternalResource.class is known to exist in a subdirectory of /org/junit so should be successfully copied in
         assertTrue("The container has a file that was copied in via a recursive copy from a JAR resource", results.contains("content.txt"));
     }
+
+    public static <T> Matcher<Iterable<? super T>> exactlyNItems(final int n, Matcher<? super T> elementMatcher) {
```

Comment:
Copied from stackoverflow. AssertJ has it built-in :\

Reply:
😬 sorry!